components/eam/src/physics/cam/nucleate_ice.F90

```diff
@@ -153,23 +144,19 @@ subroutine nucleati(  &
 
    ni = 0._r8
    tc = tair - 273.15_r8
+   aer_num = soot_num + dst3_num
```

Comment:
"aer_num" may be misleading as total aerosol number but actually it only contains soot and dust. Also, I guess dst3_num is only for dust at mode 3 (coarse)? Maybe change the name to something like "soot_dst3_num"

Reply:
variable name changed